src/libsync/propagateupload.cpp

```diff
@@ -521,6 +538,21 @@ void PropagateUploadFileCommon::checkResettingErrors()
     }
 }
 
+void PropagateUploadFileCommon::checkInsufficentStorageError()
```

Comment:
On the other hand, maybe the folder is on a different storage or with different quota requirement that the main quota. So we might mess the quota up.